src/shared/components/create-case-filters/create-case-filters.component.ts

```diff
@@ -42,6 +42,7 @@ export class CreateCaseFiltersComponent implements OnInit {
   constructor(
     private orderService: OrderService,
     private definitionsService: DefinitionsService,
+    private readonly sessionStorageService: SessionStorageService
```

Comment:
_Nitpick:_
Can you make the two above _also_ `readonly`, please? :)

Reply:
done